vocab/credentials/v2/vocabulary.yml

```diff
@@ -45,56 +50,56 @@ class:
     label: Credential evidence
     comment: A Credential Evidence scheme provides enough information to a verifier to determine whether the evidence gathered meets their requirements for issuing a credential. The precise content of each evidence scheme is determined by the specific evidence type definition.
 
+  - id: CredentialGraph
+    label: (Verifiable) credential graph
+    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one <a href="#Credential">Credential</a> (usually a <a href="#VerifiableCredential">Verifiable Credential</a>).
+
   - id: VerifiableCredential
     label: Verifiable credential
     comment: |
-      A Credential is a set of one or more claims made by an issuer. A Verifiable Credential is a tamper-evident credential that has authorship that can be cryptographically verified. Verifiable Credentials can be used to build Verifiable Presentations, which can also be cryptographically verified.
-
-  - id: VerifiableCredentialGraph
-    label: Verifiable credential graph
-    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one Verifiable Credential
+      A Verifiable Credential is a <a href="#Credential">Credential</a> that is tamper-evident and has authorship that can be cryptographically verified. Verifiable Credentials can be used to build Verifiable Presentations, which can also be cryptographically verified.
 
   - id: VerifiablePresentation
     label: Verifiable presentation
     comment: |
-      A Presentation is data derived from one or more Credentials, issued by one or more `issuers`, that is shared with a specific `verifier`. A Verifiable Presentation is a tamper-evident Presentation encoded in such a way that authorship of the data can be trusted after a process of cryptographic verification. Certain types of verifiable presentations might contain data that is synthesized from, but do not contain, the original verifiable credentials (for example, zero-knowledge proofs).
+      A Presentation is data derived from one or more <a href="#Credential">Credentials</a> (usually, but not necessarily, <a href="#VerifiableCredential">Verifiable Credentials</a>), issued by one or more `issuers`, that is shared with a specific `verifier`. A Verifiable Presentation is a tamper-evident Presentation encoded in such a way that authorship of the data can be trusted after a process of cryptographic verification. Certain types of verifiable presentations might contain data that is synthesized from, but do not contain, the original verifiable credentials (for example, zero-knowledge proofs).
```

Comment:
This is wrong, presentations are made from "VerifiableCredentials"... not "Credentials".

<img width="856" alt="Screen Shot 2023-03-03 at 9 52 05 AM" src="https://user-images.githubusercontent.com/8295856/222765557-68108ed4-dc04-4394-9d76-56af33770e5a.png">

- https://w3c.github.io/vc-data-model/#presentations

Reply:
> [@OR13] This is wrong, presentations are made from "VerifiableCredentials"... not "Credentials".

The block you quoted (by screencap, which is rather hard to work with, and really should be copied-and-pasted text, still including the link to the original) speaks of _VERIFIABLE presentations_, not _presentations_.

PLEASE stop acting as if quite specific words and phrases (which were carefully and painfully arrived at, as I know from being a participant at the time) were meant to have less specific meaning, as I will continue to tell you, they were not, and your disingenuous quotations will only make the current process longer and more painful!

Reply:
@TallTed there is no RDF definition for "Presentation", and the section I am commenting on is regarding the id "VerifiablePresentation".

Reply:
Yes indeed, I had to stop on how I would have to describe a `VerifiablePresentation` and, indeed, the "logical" approach is to refer to Verifiable Presentations only. 

However. Do we have cases when a Presentation structure is used for Credentials only? If I encapsulate a Presentation in JWT with the proof left as external, I would indeed present a credential and not a Verifiable credential, right (I am not a JWT expert)? If so, do we want to go down the (painful) route of differentiating between a `Presentation` and a `VerifiablePresentation`? Isn't that an overkill? So I ended up doing what I did, as described in the PR description above.

Reply:
@TallTed I do not agree that Orie is acting disingenuously. If you have concerns that WG members are acting in bad faith, the appropriate step is to contact the chairs for a conversation. Making accusations in github comments is not appropriate.